### ISSUE TITLE
Update usage samples/pip

### ISSUE THREAD

**Issue by LinqLover** [trevorhobenshield/twitter-api-client#36]
The samples in the readme seem not to match the pip package.

Some examples:

```
>>> account = Account(email, username, password, debug=2, save=True)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: Account.__init__() got an unexpected keyword argument 'debug'

>>> scraper = Scraper(email, username, password, debug=1, save=True)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: Scraper.__init__() got an unexpected keyword argument 'debug'

>>> from twitter.search import Search
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name 'Search' from 'twitter.search' (/usr/local/lib/python3.10/dist-packages/twitter/search.py)
```

If I install the package from the latest revision of this repo, the examples (technically) work, however! So maybe there should also just be any note that the pip package is not always up to date. 😅

**Comment by trevorhobenshield** [comment]
Thanks for the feedback. Make sure you have the latest version `0.7.7`

```bash
pip freeze | grep twitter-api-client
```

if not:
```
pip uninstall twitter-api-client -y
pip install twitter-api-client --no-cache-dir
```

**Comment by LinqLover** [comment]
Okay, so I was confused because for python 3.10.6 (which is shipped by default on latest ubuntu), only v0.5.7 is on pip 😅
